### PR TITLE
Tools: Add ui-generator source code

### PR DESCRIPTION
**Pull request by jronak**

### CODE REVIEW

**Comment by dilipzotha** [comment]
Can you add .DS_Store file to .gitignore?

File: tools/ui-generator/package.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "sketch2dui",
```
Comment (dilipzotha) [inline tools/ui-generator/package.json]: Can you rename this to presto-ui-generator

File: tools/ui-generator/package.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  "version": "0.0.1",
```
Comment (dilipzotha) [inline tools/ui-generator/package.json]: Can you change version to v0.1.0

File: tools/ui-generator/package.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  "description": "Converts sketch exports to DUI",
```
Comment (dilipzotha) [inline tools/ui-generator/package.json]: Converts sketch exports to Presto-UI code

**Comment by jronak** [comment]
@dilipzotha done

File: tools/ui-generator/presto-sample/README.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+# install
```
Comment (dilipzotha) [inline tools/ui-generator/presto-sample/README.md]: Is this relevant? Can we stick to only webapp for now.

File: tools/ui-generator/presto-sample/package.json [hunk 5]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "presto-sample",
+  "version": "0.0.0",
```
Comment (dilipzotha) [inline tools/ui-generator/presto-sample/package.json]: 0.1.0

File: tools/ui-generator/presto-sample/package.json [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+  "private": false,
+  "scripts": {
+    "compile": "webpack --progress --colors --optimize-minimize",
+    "start": "node ./node_modules/webpack-dev-server/bin/webpack-dev-server.js --content-base dist/"
+  },
+  "dependencies": {
+    "axios": "^0.16.2",
+    "qr-image": "^3.2.0",
```
Comment (dilipzotha) [inline tools/ui-generator/presto-sample/package.json]: Is this needed?

File: tools/ui-generator/presto-sample/README.md (hunk 4 above)
Comment (jronak) [reply tools/ui-generator/presto-sample/README.md]: Yup its not, that's carried over from DUI README.

File: tools/ui-generator/presto-sample/package.json (hunk 6 above)
Comment (jronak) [reply tools/ui-generator/presto-sample/package.json]: Wait. No

Comment (jronak) [reply tools/ui-generator/presto-sample/package.json]: Mystique requires this, https://github.com/juspay/presto-ui/blob/master/core/src/WEB/JBridgeInterface.js#L29.

Will send out another PR to cleanup presto-ui/core

**Comment by jronak** [comment]
Updated

**Pull request merged**